src/components/dls/InlineWordByWord/index.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+import classNames from 'classnames';
+import { shallowEqual, useSelector } from 'react-redux';
+
+import styles from './InlineWordByWord.module.scss';
+
+import { selectWordByWordFontScale } from '@/redux/slices/QuranReader/styles';
+
+const FONT_SIZE_CLASS_MAP = {
+  1: styles.xs,
+  2: styles.sm,
+  3: styles.md,
+  4: styles.lg,
+  5: styles.xl,
+  6: styles.xxl,
+};
+
+type Props = {
+  word?: string;
```

Comment:
nit: I think we should rename this prop to `text` since it's not a word and can be multiple words.

Reply:
Agree will update.